conda/common/logic.py

```diff
@@ -217,7 +217,7 @@ def process_solution(self, sat_solution):
         return sat_solution
 
 
-class CryptoMiniSatSolver(SatSolver):
+class _PyCryptoSatSolver(SatSolver):
```

Comment:
PyCryptoSatSolver is more consistent/makes more sense to use here. (In future `cryptominisat` might be an additional, different choice for `context.sat_solver` that would indicate direct usage of `libcryptominisat`.)